pandas/core/generic.py

```diff
@@ -10133,9 +10133,13 @@ def tz_convert(
         """
         Convert tz-aware axis to target time zone.
 
+        To convert to UTC and get a tz-naive axis, pass tz=None.
```

Comment:
Please only mention this once, not 3 times

Reply:
Yeah mentioning in this in the parameters should be sufficient.

Moreover, I think it would be even better if this was shown in the `Examples` section

Reply:
updated this PR.

remove the function-level doc to avoid duplication.
keep parameter documentation for tz_convert and tz_localize.